src/app/body/all-products/filters/rating-filter/rating-filter.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="filter_brand">
+  <label>
+    <img src="{{starFull}}" alt="">
+    <img src="{{starFull}}" alt="">
+    <img src="{{starFull}}" alt="">
+    <img src="{{starFull}}" alt="">
+    <img src="{{starFull}}" alt="">
+    <input type="checkbox" />
+    <span></span>
+  </label>
+</div>
+
+<div class="filter_brand">
+  <label>
+    <img src="{{star}}" alt="">
+    <img src="{{starFull}}" alt="">
+    <img src="{{starFull}}" alt="">
+    <img src="{{starFull}}" alt="">
+    <img src="{{starFull}}" alt="">
+    <input type="checkbox" />
+    <span></span>
+  </label>
+</div>
+
+<div class="filter_brand">
+  <label>
+    <img src="{{star}}" alt="">
```

Comment:
Better to write:
`<img [src]="star" alt="">`

Reply:
fixed